styles.css

```diff
@@ -0,0 +1,343 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --font-family-body: 'Poppins', sans-serif;
+  --ff-headings: 'Playfair Display', serif;
+  --font-size-h1: clamp(1.5rem, -0.25rem + 20vw, 6rem);
+  --gap-header: clamp(12rem, 100vw, 25rem);
+  --button-SC-width: min(90vw, 700px);
+  --adaptive-sizing-section-2: min(100vw, 700px);
+  --fs-body: clamp(1rem, -0.5rem + 6.4vw, 1.25rem);
+  --ff-project-name-goes-here: 'Segoe UI', tahoma, geneva, verdana, sans-serif, 'Poppins';
+  --fs-project-name-goes-here: min(8vw, 70px);
+  --margin-top-project-name-goes-here: clamp(1rem, 1rem + 0vw, 1rem);
+  --margin-left-project-name-goes-here: clamp(1rem, 1rem + 0vw, 1rem);
+  --margin-bottom-project-name-goes-here: clamp(1rem, -1.4rem + 10.24vw, 1.8rem);
+  --font-size-languages: 4vw;
+  --margin-bottom-languages: clamp(1rem, -1.4rem + 10.24vw, 1.8rem);
+  --project-image-width: min(100vw, 700px);
+  --languages-width: min(100vw, 700px);
+  --see-this-project-width: min(80vw, 700px);
+  --fs-see-this-project: clamp(1rem, -2rem + 12.8vw, 2rem);
+  --color-boxes-title-font: 'Poppins', sans-serif;
+  --color-boes-list-font: 'Poppins', sans-serif;
+}
+
+header {
+  display: flex;
+  gap: 55vw;
+  margin-bottom: 2rem;
+}
+
+/* add media query to increase gap later */
+
+.section-1 {
+  position: relative;
+}
+
+h1 {
+  font-family: var(--ff-headings);
+  font-style: normal;
+  font-weight: 400;
+  font-size: var(--font-size-h1);
+  line-height: 100%;
+  color: #1a2236;
+  margin-left: 1rem;
+  margin-bottom: 2rem;
+}
+
+p {
+  left: 16px;
+  top: 224px;
+  font-family: var(--font-family-body);
+  font-style: normal;
+  font-weight: 400;
+  font-size: var(--fs-body);
+  letter-spacing: -0.01em;
+  color: #1a2236;
+  margin-left: 1rem;
+  margin-bottom: 2.5rem;
+  max-width: 180%;
+}
+
+.image-1 {
+  width: var(--adaptive-sizing-section-2);
+  margin-bottom: 3rem;
+}
+
+.Start-collab {
+  width: var(--button-SC-width);
+  height: 50px;
+  background-color: #1a2236;
+  color: white;
+  border-color: transparent;
+  font-family: var(--font-family-body);
+  font-size: clamp(1rem, -2rem + 12.8vw, 2rem);
+  margin-left: 1rem;
+  position: relative;
+  margin-bottom: 3rem;
+}
+
+.Start-collab:hover {
+  background-color: white;
+  color: black;
+  border-width: 1px;
+  border-color: black;
+}
+
+.project-image {
+  background-color: #000;
+  width: var(--project-image-width);
+}
+
+.languages {
+  width: var(--languages-width);
+  background: #1a2236;
+  border: 0.333333px solid rgba(255, 255, 255, 0.1);
+  backdrop-filter: blur(0.5px);
+}
+
+.section-2 {
+  margin-bottom: 3rem;
+}
```

Comment:
You have a good design. However, as mentioned in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/100a933087f033def9233c8cce29c72194391fe0/portfolio/2_mobile.md?plain=1#L51), kindly use a `display: grid` property for the container of the works section.